frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/common.unit.spec.tsx

```diff
@@ -17,24 +17,23 @@ describe("SettingsEditor", () => {
       });
 
       await userEvent.click(screen.getByText("Embedding"));
-      await userEvent.click(screen.getByText("Interactive embedding"));
-      expect(screen.queryByText("Authorized origins")).not.toBeInTheDocument();
+      expect(screen.getByText("Interactive embedding")).toBeInTheDocument();
```

Comment:
The test was wrong, you can't click this label.
![image](https://github.com/user-attachments/assets/dd7b7e7f-7983-49a0-80c2-627163646051)

We used to allow OSS users to go into the interactive settings page at one point, but we disallow that now. So we can only test what's on the interactive embedding card.